src/vl-typography.js

```diff
@@ -61,7 +61,15 @@ export class VlTypography extends vlElement(HTMLElement) {
     return observer;
   }
 
-  __processTemplateParameters(template, params) {
+  /**
+   * Dit vormt een template met placeholders voor parameters om in een tekst waarin deze placeholders vervangen
+   * zijn met hun waarden.
+   *
+   * @param {string} template De template waarin placeholders zitten in de vorm van "${parameter.parameterNaam}"
+   * @param {object} params Een object met keys die de naam van de parameter voorstellen (bv. "parameterNaam") en hun waarden
+   * @returns {string} De resulterende tekst
+   */
+  static processTemplateParameters(template, params) {
```

Comment:
zeker nu ik het zo zie staan denk ik wel "dees is eigenlijk een UTIL functie" die nu maar toevallig hier staat, en er dicteert niks dat het templating systeem hetzelfde voor vl-typography dan wel vl-proza MOET zijn.

maar dit gezegd zijnde is het natuurlijk ook niet echt een probleem ...

Reply:
't is dat proza momenteel onderliggend gebruikmaakt van typography om deze functionaliteit aan te bieden als je via attributen op de proza-message tag werkt dat het me op zich wel ok lijkt om het zo te houden. 
Maar dat hoeft natuurlijk niet zo te blijven, da's inner workings van proza eigenlijk en kan vervangen worden met een aanroep naar processTemplateParameters.

Reply:
Ik had dit even met @SanderKleykens besproken. Het is inderdaad een util functie en kan (bijvoorbeeld) verplaatst worden naar de vl-ui-core al zijn er momenteel maar 2 afnemers en zie ik niet meteen een extra afnemer van deze functie. Het leek me daarom niet de moeite om de logica te verplaatsen en de testen te gaan isoleren. Er is momenteel een relatie tussen vl-ui-proza-message en vl-ui-typography om parameters te verwerken dus die relatie is hierbij niet geïntroduceerd dan wel versterkt en lijkt me geen probleem.